packages/input-number/src/input-number.vue

```diff
@@ -123,9 +136,17 @@
       maxDisabled() {
         return this._increase(this.value, this.step) > this.max;
       },
-      precision() {
-        const { value, step, getPrecision } = this;
-        return Math.max(getPrecision(value), getPrecision(step));
+      numPrecision() {
+        const { value, step, getPrecision, precision } = this;
+        const stepPrecision = getPrecision(step);
+        if (this.precision !== undefined) {
+          if (stepPrecision > this.precision) {
+            console.warn(`the precision of step is ${stepPrecision}, it's bigger than the percision in props which is ${precision}`);
```

Comment:
`[Element Warn][InputNumber]precision should not be less than the decimal places of step`